editoast/src/views/mod.rs

```diff
@@ -1,4 +1,5 @@
 mod authz;
+pub mod core_error_response;
```

Comment:
```suggestion
mod core_error_response;
use core_error_response::CoreErrorResponse;
```

avoids repetition in import paths

Reply:
Done.